tests/net/err.rs

```diff
@@ -41,7 +41,12 @@ where
     /// The execution time limit has been reached or exceeded.
     TimeLimitHit(time::Duration),
     /// A `Fault` is encountered in a step of a `DistAlgorithm`.
```

Comment:
We should add to the docs here that only faults reported by correct nodes are taken into account, e.g.

```rust
    /// A `Fault` was reported by a correct node in a step of a `DistAlgorithm`.
```